0-good-advice/notes/what-i-wish-someone-had-told-me.md

```diff
@@ -0,0 +1,72 @@
+Read it yourself: https://blog.samaltman.com/what-i-wish-someone-had-told-me
+
+Tool to help you apply it to your life: https://chat.openai.com/g/g-uPMOViqnT-altman-advisor
+
+
+## Applying Advice to Different Stages Of Business
```

Comment:
Feels like it's helpful to format it in the lifecycle stage, but how would someone know to check this later one? Fine with merging in but we need consideration of future improving of this.